packages/instantsearch-ui-components/src/components/chat/ChatMessage.tsx

```diff
@@ -256,32 +230,28 @@ export function createChatMessageComponent({
       );
     }
 
-    const Actions = () => {
+    const Actions = useMemo(() => {
```

Comment:
does it work fine if it's not memoised too? we can't use react's useMemo in a ui component.

Reply:
Removing the `useMemo` in `ChatMessage` seems fine, but the others in `ChatMessages` cause all the messages to re-render when you scroll

Reply:
I managed to fix it by moving these outside of the component:

```ts
export function createChatMessagesComponent({
  createElement,
  Fragment,
}: Renderer) {
  const DefaultMessageComponent =
    createDefaultMessageComponent<ChatMessageBase>({ createElement, Fragment });
  const DefaultLoaderComponent = createChatMessageLoaderComponent({
    createElement,
  });
  const DefaultErrorComponent = createChatMessageErrorComponent({
    createElement,
  });

  return function ChatMessages
  /* ... */
}
```